### PR TITLE
compute arr, psc

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Fixed
+
+- Incorrect values of autorelod in `CountDown::start` and `Pwm::new` and prescaler in `Delay`
```
Comment (Disasm) [inline CHANGELOG.md:12]:
```suggestion
- Incorrect values of autoreload in `CountDown::start` and `Pwm::new` and prescaler in `Delay` 
```

File: src/timer.rs [hunk 2]
```diff
@@ -351,6 +351,15 @@ where
     }
 }
 
+#[inline(always)]
+pub(crate) fn compute_arr_presc(freq: u32, clock: u32) -> (u16, u32) {
```
Comment (therealprof) [inline src/timer.rs:355]: Could this be a `const fn`?

**Comment by burrbull** [comment]
@therealprof Made some tests more. Works good as for me.

**Review by therealprof — APPROVED** [review]
LGTM, thanks.

bors r+

**Pull request merged**